src/Navigation/Navigation.js

```diff
@@ -29,6 +30,7 @@ export const Navigation = () => (
       options={({ route }) => ({ title: route.params.drill.title })}
     />
     <Stack.Screen name="TrainingListPage" component={TrainingListPage} options={{ title: 'Training sessions' }} />
+    <Stack.Screen name="ProgramsPage" component={ProgramsPage} options={{ title: 'Choose a program for your team' }} />
```

Comment:
On my phone, the title is cut ('Choose a program for your te')

Reply:
Then I can put only "Choose a program"